### PR TITLE
Open Feedback URL Depending on Browser

### PR DESCRIPTION
**Pull request by tikhsuP**
This PR provides the following fix:
The **Feedback** button in the extension and the **Write a Review** option in the **About** page now open the appropriate feedback pages depending on the browser (Chrome/Firefox).

### CODE REVIEW

File: scripts/about.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
-})
+//about.js
+
+// from 'utils.js'
+/*   global feedbackPageURL */
+const VERSION = chrome.runtime.getManifest().version
+const DATE = new Date().getFullYear()
+$('#version').text(VERSION)
+$('#year').text(DATE)
+$('#reviews-page').attr('href', feedbackPageURL)
```
Comment (anishsarangi) [inline scripts/about.js:9]: Add target='_blank' attribute also

Comment (tikhsuP) [reply scripts/about.js:9]: Thanks, was thinking of adding that in other links too.

Comment (cgorringe) [reply scripts/about.js:9]: I was planning to add target='_blank' to all of the links in the About page, but if you want to, go ahead.

Comment (tikhsuP) [reply scripts/about.js:9]: I added it in the `about.html` file. Please review : )

**Pull request merged**